### ISSUE TITLE
ChromeOS: disabling battery optimizations doesn't work

### ISSUE THREAD

**Issue by vadi2** [OxfordHCC/tracker-control-android#204]
Running TC on ChromeOS seems to work relatively OK, though I imagine it's not in the target scope. One thing is that at the start, the usual battery optimization dialog pops up:

![image](https://user-images.githubusercontent.com/110988/119855638-61705380-bf12-11eb-902f-5a8e040beab9.png)

However the next step doesn't work:

![image](https://user-images.githubusercontent.com/110988/119855600-56b5be80-bf12-11eb-8136-90107eb2fb25.png)

**Comment by kasnder** [comment]
Oops. I guess you can click “don’t ask again“ to hide the message? I’m not familiar with Chrome OS, but maybe you can find battery optimisation in its settings?

**Comment by kasnder** [comment]
Lovely to hear that TC works on Chrome OS though!

**Comment by vadi2** [comment]
As far as I see, the option does not seem to exist on ChromeOS - which would explain why it comes up with an error.

![image](https://user-images.githubusercontent.com/110988/119916181-53e3b980-bf64-11eb-96aa-f7679ab6734a.png)
![image](https://user-images.githubusercontent.com/110988/119916198-5ba35e00-bf64-11eb-8f8a-f195982a3d17.png)

**Comment by vadi2** [comment]
Yeah just one instance where it went wrong was when I quit the app, the chrome shell crashed - but that is something for another issue. For the time being, things are working out :)

![image](https://user-images.githubusercontent.com/110988/119916313-96a59180-bf64-11eb-860f-e9669e95f861.png)

**Comment by kasnder** [comment]
I see! I guess I won't fix this anytime soon, since you can just hide the notification, and Chrome OS doesn't seem to have the battery optimisation option.

**Comment by kasnder** [comment]
ChromeOS battery-optimization behavior is controlled by the platform/OEM and cannot be overridden by TrackerControl. Closing as an external platform limitation.
